terraform/catalogue_api/terraform.tf

```diff
@@ -0,0 +1,47 @@
+terraform {
+  backend "s3" {
+    role_arn = "arn:aws:iam::756629837203:role/catalogue-developer"
+
+    bucket         = "wellcomecollection-catalogue-infra-delta"
+    key            = "terraform/catalogue/api/catalogue_api.tfstate"
+    dynamodb_table = "terraform-locktable"
+    region         = "eu-west-1"
+  }
+}
+
+data "terraform_remote_state" "accounts_catalogue" {
+  backend = "s3"
+
+  config = {
+    role_arn = "arn:aws:iam::760097843905:role/platform-read_only"
+
+    bucket = "wellcomecollection-platform-infra"
+    key    = "terraform/platform-infrastructure/accounts/catalogue.tfstate"
+    region = "eu-west-1"
+  }
+}
+
+data "terraform_remote_state" "infra_critical" {
+  backend = "s3"
+
+  config = {
+    role_arn = "arn:aws:iam::760097843905:role/platform-read_only"
+
+    bucket = "wellcomecollection-platform-infra"
+    key    = "terraform/platform-infrastructure/shared.tfstate"
+    region = "eu-west-1"
+  }
+}
+
+// TODO remove this, hold the resources in this module
```

Comment:
You can also hold the state here by importing it - as long a we remember to remove the state rather than delete the ECR repositories in the other stack.

Reply:
This is (I think) what I meant by this comment, I was thinking I'd shift all the remaining state over when I come to delete the old stack

Reply:
I meant you could do it now, and then deal with the deleting the duplicated state from the other stack later - but either way is fine.